containers/docker/pwpush/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# pwpush-ephemeral
+FROM ruby:3.2-alpine AS build-env
+
+LABEL maintainer='pglombardo@hey.com'
+
+# Required packages
+RUN apk upgrade --no-cache \ 
```

Comment:
Not a believe of `apt upgrade` or `apk upgrade`.  It makes the image lest reproducible.

Reply:
Read a bit about you suggestion and I agree.
I haven't considered your point until now

For doc pupose: [](https://stackoverflow.com/questions/72045168/is-it-necessary-to-run-apk-update-apk-upgrade-in-a-docker-build-stage)